core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/EventV4Controller.java

```diff
@@ -46,6 +46,9 @@
     @Inject
     private StackService stackService;
```

Comment:
I think this is not used anymore, could you remove?

Reply:
removed